relayer/naive-strategy.go

```diff
@@ -140,6 +144,45 @@ func UnrelayedSequences(ctx context.Context, src, dst *Chain, srcChannel *chanty
 		return nil, err
 	}
 
+	// If this is an UNORDERED channel we can return at this point.
+	if srcChannel.Ordering != chantypes.ORDERED {
+		rs.Src = srcUnreceivedPackets
+		rs.Dst = dstUnreceivedPackets
+		return rs, nil
+	}
+
+	// For ordered channels we want to only relay the packet whose sequence number is equal to
+	// the expected next packet receive sequence from the counterparty.
+	if len(srcUnreceivedPackets) > 0 {
+		nextSeqResp, err := dst.ChainProvider.QueryNextSeqRecv(ctx, dsth, srcChannel.Counterparty.ChannelId, srcChannel.Counterparty.PortId)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, seq := range srcUnreceivedPackets {
+			if seq == nextSeqResp.NextSequenceReceive {
+				rs.Src = append(rs.Src, seq)
+				break
+			}
+		}
+	}
+
+	// For ordered channels we want to only relay the packet whose sequence number is equal to
+	// the expected next packet receive sequence from the counterparty.
```

Comment:
Should this comment be different from the above one on L154? They look the same.

Reply:
Both of these blocks of code do the same thing just operating on different chains, idk if I meant to add the comment twice but I think having it once is sufficient. Thanks for pointing that out!